firefox-install-demo.toml

```diff
@@ -78,3 +122,38 @@ from_expression = """
 """
 submission_date_column = "submission_date"
 client_id_column = "document_id"
+
+[data_sources.firefox_installs_all_versions]
+from_expression = """
+    (
+        WITH success_rate_per_branch AS (
+            SELECT 
+                DATE(submission_timestamp) AS submission_date,
+                document_id,
+                build_channel,
+                succeeded,
+                "All versions" AS branch,
```

Comment:
this part is actually only relevant when the datasource is used to define the `population`, but ignored when used to define a metric